### ISSUE TITLE
problems submitting the textarea value to an action

### ISSUE THREAD

**Issue by jfhovinne** [wymeditor/wymeditor#117]
First, I couldn't find any examples which actually submitted the textarea value somewhere -- I assume that adding 'name="foo"' to the textarea element and adding an action attribute value to the form element should work;  it does to some extent.

I've verified the following behavior on Firefox 3.0.1 and MSIE7, and I've verified this behavior against trunk and 0.5:

When I type text in the editor, and then press submit, no text is sent to my action.

When I type text, then click the "HTML" button at least once, and then submit, the text I've entered is sent to my action.

When I type text, then click the "HTML" button, then type more text, and then submit, only the original text is sent to the action.

In other words, the only way to submit text to the action is to click on the HTML button before submitting.

Am I missing something?
